cherro-front/package.json

```diff
@@ -37,7 +37,12 @@
     ]
   },
   "jest": {
-    "clearMocks": true
+    "clearMocks": true,
+    "coverageThreshold": {
```

Comment:
I think this makes sense, but why did you add it?

Reply:
it was one of the possible solutions to the test problem

Reply:
Oh, then let's remove it 😄 Unless you anticipate any issues